end-to-end-tests/docker-compose.yaml

```diff
@@ -0,0 +1,85 @@
+version: '2.4'
+services:
+  anchor-platform-server:
+    image: anchor-platform
+    build:
+      context: ../../
+      dockerfile: integration-tests/Dockerfile
```

Comment:
Does this mean we need to build docker image every time we run?

Reply:
I removed that, but docker won't build a new image if `anchor-platform` was found. The `build` configs are used only in the case when the image is not found

Reply:
Besides that when I try to pull anchor image from Docker registry it fails because I don't have access 😢